tests/cupyx_tests/scipy_tests/ndimage_tests/test_filters.py

```diff
@@ -407,15 +416,15 @@ def test_filter(self, xp, scp):
             'filter': ['maximum_filter', 'minimum_filter'],
             'origin': [0, (0, 1, 1, 0)],
             'kshape': [(3, 5, 3, 5)],
-            'mode': ['reflect'],
+            'mode': ['constant'],
             'cval': [0, 2],
             'footprint': [True],
         }) +
         testing.product({
             'filter': ['uniform_filter'],
             'origin': [0, (0, 1, 1, 0)],
             'kshape': [(3, 5, 3, 5)],
-            'mode': [('reflect', 'nearest', 'mirror', 'wrap')],
+            'mode': [('reflect', 'constant', 'mirror', 'nearest')],
```

Comment:
Ditto (regarding the filter support for `wrap`)

Reply:
It is still supported, not sure why I had removed. will add it back here